zproject/urls.py

```diff
@@ -718,10 +716,10 @@
     path("calls/zoom/deauthorize", deauthorize_zoom_user),
     # Used to join a BigBlueButton video call
     path("calls/bigbluebutton/join", join_bigbluebutton),
-    # API and integrations documentation
-    path("integrations/doc-html/<integration_name>", integration_doc),
-    path("integrations/", integrations_view),
-    path("integrations/<path:path>", integrations_view),
+    # Integrations documentation
+    path("integrations/doc/<str:integration_name>", integrations_doc),
+    path("integrations/", integrations_catalog, {"category_slug": "all"}),
+    path("integrations/<str:category_slug>", integrations_catalog),
```

Comment:
I think I want `/integrations/category/{category_slog}` and `/integrations/{integration_name}` as the URLs -- the /doc/ prefix isn't great.